units/modprobe@.service

```diff
@@ -16,4 +16,5 @@ ConditionCapability=CAP_SYS_MODULE
 
 [Service]
 Type=oneshot
+RemainAfterExit=yes
```

Comment:
Have you read the discussion in #14819?. I am not sure this is safe or not...

Reply:
Oh, I forgot about about that PR. I'll drop my patch then. Let's continue the discussion there.